staging/src/k8s.io/component-base/go.mod

```diff
@@ -28,6 +28,7 @@ require (
 	k8s.io/client-go v0.0.0
 	k8s.io/klog/v2 v2.60.1
 	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
+	sigs.k8s.io/yaml v1.2.0
```

Comment:
Makes sense.

However, sigs.k8s.io/yaml was (and remains) an indirect dependency because of:
```
k8s.io/component-base/codec
k8s.io/apimachinery/pkg/runtime/serializer
k8s.io/apimachinery/pkg/runtime/serializer/json
sigs.k8s.io/yaml
```